elfi/methods/posteriors.py

```diff
@@ -24,15 +24,31 @@ class BolfiPosterior:
 
     Note that when using a log discrepancy, h should become log(h).
 
+    ALTERNATIVELY
```

Comment:
Create a new class:
`ExponentialBOLFIPosterior(BOLFIPosterior)` and override methods as required.